src/scss/settings/_whoWeHelp.scss

```diff
@@ -0,0 +1,137 @@
+section.whoWeHelp {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 1015px;
+
+    div.whoWeHelp-intro {
+        margin-top: 70px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        h1 {
+            font-size: 38px;
+            font-weight: normal;
+            color: $black-ish;
```

Comment:
More along patterns would be create separate black colours like $black-1, $black-2